gobblin-cluster/src/test/java/gobblin/cluster/GobblinClusterKillTest.java

```diff
@@ -107,9 +108,11 @@ private void setupTestDir() throws IOException {
    */
   private void setupManager(int id) throws Exception {
     _clusterManagers[id] =
-        new GobblinClusterManager(TestHelper.TEST_APPLICATION_NAME, TestHelper.TEST_APPLICATION_ID,
-            _config.withValue(GobblinClusterConfigurationKeys.HELIX_INSTANCE_NAME_KEY,
-                ConfigValueFactory.fromAnyRef("Manager_" + id)),
+        new GobblinClusterManager(TestHelper.TEST_APPLICATION_NAME, TestHelper.TEST_APPLICATION_ID, _config
+                .withValue(GobblinClusterConfigurationKeys.HELIX_INSTANCE_NAME_KEY,
+                        ConfigValueFactory.fromAnyRef("Manager_" + id))
+                .withValue(ConfigurationKeys.STATE_STORE_ROOT_DIR_KEY,
```

Comment:
Good call.  I'm not sure how to tell if the state store is enabled. Ideally, each state store implementation should have a corresponding cleaner that would be automatically included.  I think for now it would be best to just not automatically start the state store cleaner.